config/src/main/java/com/typesafe/config/impl/ConfigImplUtil.java

```diff
@@ -32,6 +32,14 @@ else if (a == b) // catches null == null plus optimizes identity case
             return a.equals(b);
     }
 
+    public static boolean isC0Control(final char ch) {
```

Comment:
do we ever use the "char" version or only the codepoint one? It's probably a bug if we use the char one.